<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginUI</class>
 <widget class="QWidget" name="LoginUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userIdEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="registrationLink">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Need registration ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>Failed to log in</string>
   </property>
  </widget>
  <widget class="QComboBox" name="userTypeBox">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>160</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Clinician</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Family Physician</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Patient</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User type:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
